public/video-ui/src/components/FormFields/TagPicker.js

```diff
@@ -392,6 +406,11 @@ export default class TagPicker extends React.Component {
           {this.props.notification.message}
           </p>
             : ''}
+        {hasError
```

Comment:
I wonder if using the logical `&&` here is cleaner than the ternary: 

```js
hasError && <p className="form__message form__message--error">
           {this.props.notification.message}
           </p>
```

Though it looks as if there's a consistency of using ternary operators with empty string in else, so not major

Reply:
I'll leave these alone for the reason you stated.